### ISSUE TITLE
Test Event Logs Sub Menu Log

### ISSUE THREAD

**Issue by justin0309** [openbmc/openbmc-test-automation#2254]
Is it because the test code runs too fast that the web page has no time to respond?
rebot test(v4.0-stable)
1.Select And Verify Default UTC Timezone For Events
![image](https://github.com/user-attachments/assets/6a8a27ee-c9bd-41cd-8eea-4691c24c92fd)

**Comment by gkeishin** [comment]
@rahulmah Can u check on this and respond

**Comment by rahulmah** [comment]
Yes @justin0309 We should actually be waiting for event log menu to show up before clicking on it here.

**Comment by gkeishin** [comment]
Closing this ticket.
